File: spark-streaming-kafka-demo/src/main/scala/com/phicomm/demo/scala/DemoMain.scala
package com.phicomm.demo.scala

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.context.annotation.ComponentScan

@SpringBootApplication
@ComponentScan(Array(
  "com.phicomm.demo.scala.runner"
))
class DemoMain

object DemoMain {

  def main(args: Array[String]): Unit = {
    SpringApplication.run(classOf[DemoMain])
  }

}

File: logger-roller-test/src/main/java/me/codetalk/demo/DemoMain.java
package me.codetalk.demo;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.ComponentScan;

/**
 * Created by guobiao.xu on 2018/7/2.
 */
@SpringBootApplication
@ComponentScan(basePackages = {
        "me.codetalk.demo.runner"
})
public class DemoMain {

    public static void main(String[] args) {
        SpringApplication.run(DemoMain.class);
    }

}
